spec.html

```diff
@@ -28544,7 +28544,8 @@
             </tbody>
           </table>
         </figure>
-        <p>The production <emu-grammar>CharacterClassEscape :: `W`</emu-grammar> evaluates by returning the set of all characters not included in the set returned by <emu-grammar>CharacterClassEscape :: `w`</emu-grammar> .</p>
+        <p>and the Boolean *false*.</p>
```

Comment:
I'm open to suggestions for a different place to add the "and the Boolean *false*." then after the table.